pkg/util/ovs/parse.go

```diff
@@ -235,6 +235,7 @@ func ParseFlow(ptype ParseType, flow string, args ...interface{}) (*OvsFlow, err
 		}
 		flow = flow[end:]
 	}
+	flow = origFlow
```

Comment:
(Unrelated fix: the parsing slowly eats away `flow` so that by the end of the loop it's `""`, but the error messages below all included `flow` as though it was still useful. So this just fixes that.)